modoboa_installer/database.py

```diff
@@ -150,6 +152,9 @@ def create_user(self, name, password):
         self._exec_query(
             "CREATE USER '{}'@'%' IDENTIFIED BY '{}'".format(
                 name, password))
+        self._exec_query(
+            "CREATE USER '{}'@'localhost' IDENTIFIED BY '{}'".format(
+                name, password))
```

Comment:
I thought '%' was a wildcard to allow connections frow anywhere? Why do you need to add localhost?